<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>409</width>
    <height>443</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>MV Boli</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>NVDI</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(254, 254, 253);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lblInfraRojo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">selection-background-color: rgb(85, 255, 255);</string>
    </property>
    <property name="text">
     <string>Banda Infra Roja</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblRojo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Banda Roja</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtInfr_roja">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>100</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>131</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>131</width>
      <height>20</height>
     </size>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;justify&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">border-top-color: rgb(0, 85, 0);
border-bottom-color: rgb(229, 255, 215);
border-left-color: rgb(0, 85, 0);
border-right-color: rgb(0, 85, 0);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnConvertir">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>110</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(195, 154, 122);
background-color: rgb(142, 187, 167);</string>
    </property>
    <property name="text">
     <string>Calcular</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSalir">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>380</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(205, 198, 195);
background-color: rgb(142, 187, 167);</string>
    </property>
    <property name="text">
     <string>Salir</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLimpiar">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>380</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(205, 198, 195);
background-color: rgb(142, 187, 167);
</string>
    </property>
    <property name="text">
     <string>Limpiar</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtRoja">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>130</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtNDVI">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>180</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lblNDVI">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>NDVI</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblinter">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Interpretaci√≥n</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblcalculadora">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Calligraphy</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string>Calculadora NDVI</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>401</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QTextEdit" name="txt_interp">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>210</y>
      <width>231</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_normal">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>NDVI Normalizado</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnNormal">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>320</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(142, 187, 167);</string>
    </property>
    <property name="text">
     <string>Normalizar</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_ndvi8b">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>320</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-top-color: rgb(0, 85, 0);
border-bottom-color: rgb(229, 255, 215);
border-left-color: rgb(0, 85, 0);
border-right-color: rgb(0, 85, 0);</string>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lblformula">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>180</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Narrow</family>
      <pointsize>8</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">selection-background-color: rgb(85, 255, 255);</string>
    </property>
    <property name="text">
     <string>NDVI = (NIR-Red) / (NIR+Red)</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>409</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
